docs/features/font-size.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Font size
+category: features
+---
+
+{@snippet features/build-font-size-source}
+
+The {@link module:font/fontsize~FontSize} feature enables support for setting font size.
+
+## Demo
+
+{@snippet features/font-size}
+
+## Configuring font size options
+
+It is possible to configure which font size options are supported by the editor. Use the {@link module:font/fontsize~FontSizeConfig#options `fontSize.options`} configuration option to do so.
+
+Use the special keyword `'normal'` to use the default font size defined in the web page styles — it disables the font-size feature.
+
+The font size feature supports two ways of defining configuration: using  predefined (named) presets or simple numeric values.
+
+### Configuration using the predefined named presets
+
+The font size feature defines 4 named presets:
+- `'tiny'`
+- `'small'`
+- `'big'`
+- `'huge'`
+
+Each size is represented in the view as a `<span>` element with the `text-*` class. For example, the `'tiny'` preset looks as follows in the editor data:
```

Comment:
We must clearly state that we don't provide the styles. It's up to developers to style `.ck-content span.tiny {}` and we must show a short snippet how to do that.